app.py

```diff
@@ -117,7 +118,7 @@ def login():
                 else:
                     return "error"
                 flash("You are now login", "success")
-                return redirect(url_for('home'))
+                return redirect(url_for('upload'))
             else:
                 flash("Incorrect password", "danger")
                 return render_template("login.html")
```

Comment:
hi, it is already there if user login, he would be redirected to the upload route.